app/src/main/java/com/arctouch/codechallenge/model/Movie.java

```diff
@@ -2,9 +2,10 @@
 
 import com.squareup.moshi.Json;
 
+import java.io.Serializable;
 import java.util.List;
 
-public class Movie {
+public class Movie implements Serializable {
```

Comment:
Why did you choose to use Serializable instead of Parcelable?

Reply:
Honestly I've used Serializable because it's simpler to implement, and there's no need to implement other methods. But, I know that Parcelable has a better performance, is faster and better than Serializable.

Reply:
I see. Have you heard about Parceler? It's a great library that implements all boilerplate code regarding `Parcelable` under the hood, using annotation processor: https://github.com/johncarl81/parceler

And for Kotlin, we have the Parcelize one:
https://android.jlelse.eu/yet-another-awesome-kotlin-feature-parcelize-5439718ba220

They're both very known on Android community and is very straightforward to implement. I suggest to take an eye out on both and have a try for your further (side) projects.